src/lib/es2015.symbol.wellknown.d.ts

```diff
@@ -124,6 +124,15 @@ interface WeakMap<K, V>{
     readonly [Symbol.toStringTag]: "WeakMap";
 }
 
+interface Set<T> {
+    [Symbol.iterator]():IterableIterator<T>;

```

Comment:
Iterable definition depends on Symbol.Iterator. it is rather meaningless to have iterables and not the definition of Symbol.Iterator.

the other ones are self contained, you can chose to use them or not.